### ISSUE TITLE
Ajax error when configuring panel on page after creating a new text panel entity

### ISSUE THREAD

**Issue by StephenOTT** [wet-boew/wet-boew-drupal#121]
Getting the follow message box error after the following steps:
1. Go to a Basic Page
2. Select Configure This Page
3. Create a New Panel
4. In the new panel creation window select the Text Entity window provide a Title and content text.
5. Save Panel and Save Page.
6. On the same page select the Configure this Page button again
7. Select the Configure Gears for a panel.  The error message below should appear.

An AJAX HTTP error occurred.
HTTP Result Code: 200
Debugging information follows.
Path: /panels/ajax/ipe/edit-pane/panelizer%3Anode%3A6%3Apage_manager/new-3
StatusText: OK
ResponseText: [{"command":"settings","settings":{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"wetkit_adaptivetheme","theme_token":"kn1AEYc4Dg2HQWa9bvVmM9Utdc62Y9Aw-xSYE2dfsKY"},"overlay":{"paths":{"admin":"file\/_\/edit\nfile\/_\/delete\nuser\/_\/linkchecker\nlinkchecker\/_\/edit\nmedia\/_\/edit\/_\noverlay\/dismiss-message\nuser\/_\/shortcuts\nadmin\nadmin\/_\nbatch\ntaxonomy\/term\/_\/edit\nuser\/_\/cancel\nuser\/_\/edit\nuser\/_\/edit\/_\ntaxonomy\/_\/translate\ntaxonomy\/_\/translate\/_\nadmin\/structure\/panels\/entity\/view\/_\/panelizer_\nnode\/_\/panelizer_\ntaxonomy\/term\/_\/panelizer_\nuser\/_\/panelizer_\nnode\/_\/revisions\/view\/_\/_","non_admin":"admin\/structure\/block\/demo\/_\nadmin\/apps\/_\/app\/_\/enable\nadmin\/apps\/_\/app\/_\/disable\nadmin\/reports\/status\/php"},"pathPrefixes":["fr"],"ajaxCallback":"overlay-ajax"},"CToolsModal":{"modalSize":{"type":"scale","width":".8","height":".8","addWidth":0,"addHeight":0,"contentRight":25,"contentBottom":75},"modalOptions":{"opacity":".55","background-color":"#FFF"},"animationSpeed":"fast","modalTheme":"CToolsModalDialog","throbberTheme":"CToolsModalThrobber"}},"merge":true},{"command":"modal_display","title":"Error","output":"Invalid pane id."}][{"command":"settings","settings":{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"wetkit_adaptivetheme","theme_token":"kn1AEYc4Dg2HQWa9bvVmM9Utdc62Y9Aw-xSYE2dfsKY"},"overlay":{"paths":{"admin":"file\/_\/edit\nfile\/_\/delete\nuser\/_\/linkchecker\nlinkchecker\/_\/edit\nmedia\/_\/edit\/_\noverlay\/dismiss-message\nuser\/_\/shortcuts\nadmin\nadmin\/_\nbatch\ntaxonomy\/term\/_\/edit\nuser\/_\/cancel\nuser\/_\/edit\nuser\/_\/edit\/_\ntaxonomy\/_\/translate\ntaxonomy\/_\/translate\/_\nadmin\/structure\/panels\/entity\/view\/_\/panelizer_\nnode\/_\/panelizer_\ntaxonomy\/term\/_\/panelizer_\nuser\/_\/panelizer_\nnode\/_\/revisions\/view\/_\/_","non_admin":"admin\/structure\/block\/demo\/_\nadmin\/apps\/_\/app\/_\/enable\nadmin\/apps\/_\/app\/_\/disable\nadmin\/reports\/status\/php"},"pathPrefixes":["fr"],"ajaxCallback":"overlay-ajax"},"CToolsModal":{"modalSize":{"type":"scale","width":".8","height":".8","addWidth":0,"addHeight":0,"contentRight":25,"contentBottom":75},"modalOptions":{"opacity":".55","background-color":"#FFF"},"animationSpeed":"fast","modalTheme":"CToolsModalDialog","throbberTheme":"CToolsModalThrobber"}},"merge":true},{"command":"modal_dismiss"}]

**Comment by thriuin** [comment]
I also ran into an Ajax issue when I was attempting to add a new action in the Rules module. I was able to get around this by temporarily setting jQuery to 1.5 (Configuration->Development->jQuery update). Unfortunately, I was able to reproduce the error described here even with jQuery 1.5. 

However, after navigating away from the page, and then returning, clicking on the configure gear icon seems to work and the error does not occur.
